istresearch/rapidpro
# P4-2334 - Add Instagram and Facebook chat PM modes and icon for IG

# For the Reviewer

When this is complete, you should approve the PR via github.



## Summary
Added PM instagram and Facebook modes

#### Release Note
- Added PM instagram and Facebook modes
- Alphabetized the postmaster identifiers in the contact urn dropdown. 

#### Breaking Changes
None.

## Quality Assurance

You have gathered the following items:

Given all of the items above, you have updated your Application ORT at the following locations:
- **Features and Alerting**: <link to app ORT>Required.
- **P4 Alerting**: <link to p4-alerting PR>Required.

## Testing and Verification

1. Create a contact and assign a `postmaster instagram identifier` scheme to it, with any value. Confirm that the URN icon is the instagram logo.
2. Create a contact and assign a `postmaster facebook identifier` scheme to it, with any value. Confirm that the URN icon is the facebook logo.
3. Create postmaster channels for facebook and instagram.
